.github/container/test-jax.sh

```diff
@@ -142,6 +142,7 @@ case "${BATTERY}" in
         JOBS=$((NGPUS * JOBS_PER_GPU))
         EXTRA_FLAGS="--local_test_jobs=${JOBS} --test_env=JAX_TESTS_PER_ACCELERATOR=${JOBS_PER_GPU} --test_env=JAX_EXCLUDE_TEST_TARGETS=PmapTest.testSizeOverflow"
         BAZEL_TARGET="${BAZEL_TARGET} //tests:gpu_tests"
+        EXTRA_CONFIG_FLAG="--enable_cuda"
```

Comment:
It appears that all batteries need to be configured as `--enable_cuda`.

Reply:
Done